lab1.js

```diff
@@ -78,12 +79,33 @@ var sentence2 = 'Come over here so you can scratch my belly.';
 // 'chirp' (10 points)
 
 // your code goes here
-
+function stringSplit(sentence) {
+  // this.sentence = sentence;
+  var newSentence = sentence.split(' ');
+  var i;
+  for (i = 0; i < newSentence.length; i++) {
+    newSentence[i] = 'chirp';
+  }
+  return newSentence;
+}
+var meerkatTranslator = stringSplit(sentence1);
```

Comment:
make sure you assign the new chirp string to sentence1 - right now the assert is failing (sentence1 hasn't been changed)